src/web/lib/storage.js

```diff
@@ -85,16 +101,35 @@ function notifySelfForStorage(storageKey) {
 
 function init(store) {
   const updateSavedThemesInStore = () => {
-    const savedThemes = listThemes();
+    const savedThemes = themeStorage.list();
     log("updateSavedThemesInStore", savedThemes);
     store.dispatch(actions.ui.setSavedThemes({ savedThemes }));
   };
 
+  const loadAllImagesIntoStore = () => {
+    const items = Object.values(imageStorage.list()).reduce(
+      (acc, { data: item }) => ({ ...acc, [item.name]: item }),
+      {}
+    );
+    log("loadAllImagesIntoStore", items);
+    store.dispatch(actions.images.loadImages(items));
+  };
+
+  const loadOneImageIntoStore = (key, newValue) => {
+    const image = JSON.parse(newValue).data;
+    log("loadOneImageIntoStore", key, image, newValue);
+    store.dispatch(actions.images.loadImage(image));
+  };
+
+  loadAllImagesIntoStore();
   updateSavedThemesInStore();
 
   window.addEventListener("storage", e => {
     log("storage event", e);
-    if (isThemeStorageKey(e.key)) {
+    if (imageStorage.isKey(e.key) && e.newValue) {
```

Comment:
Trying to be careful here to ensure other tabs besides the current tab get image updates loaded into their redux stores